src/tunacode/types/callbacks.py

```
@@ -12,18 +12,51 @@
+
+
+@runtime_checkable
+class StreamResultProtocol(Protocol):
+    """Protocol for streaming result context objects.
+
+    This is passed to tool callbacks for result submission.
+    The actual implementation details are framework-specific.
+    """
+
+    # Marker protocol - concrete attributes are framework-specific
+    pass
```

Ruff (0.14.14)

[warning] Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)